Documentation/minio-object-store-crd.md

```diff
@@ -18,6 +18,14 @@ metadata:
 spec:
   scope:
     nodeCount: 4
+    volumeClaimTemplates:
```

Comment:
the CRD details section of this page should also get an entry for this new volumeClaimTemplates field.